dts/bindings/sensor/sensor-device.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Sensor Device
+
+include: base.yaml
+
+properties:
+  vendor:
```

Comment:
I'm not keen on vendor as this seems like duplicated info from the compatible of the sensor itself.